packages/client/src/pages/TreatmentsServices.jsx

```diff
@@ -415,11 +424,19 @@ function OurImplementationModel() {
 
 function ImplementationFrameworks() {
   return (
-    <Paper id="implementation-frameworks" sx={{ display: 'flex', margin: '1rem 0', padding: '2em' }}>
-      <Box sx={{ width: '40%' }}>
+    <Paper
+      id="implementation-frameworks"
+      sx={{
+        display: 'flex',
+        flexDirection: { xs: 'column', lg: 'row' },
+        margin: '1rem 0',
+        padding: { lg: '2em', xs: '1rem' }
+      }}
+    >
+      <Box sx={{ width: { lg: '40%', xs: '100%' } }}>
         <img width="100%" src={prependStrapiURL('/uploads/implementationframeworks_8afd8c3f2c.png')} />
       </Box>
```

Comment:
```suggestion
      <Box
        component="img"
        sx={{ width: { lg: '40%', xs: '50%' }, display: 'block', margin: '0 auto' }}
        src={prependStrapiURL('/uploads/implementationframeworks_8afd8c3f2c.png')}
      />
```
Mirroring what you did for `MeasurementBasedCare` since the image here would otherwise be similarly enormous when on xs - ?